server.go

```diff
@@ -3282,7 +3295,8 @@ type openChanReq struct {
 // connection is established, or the initial handshake process fails.
 //
 // NOTE: This function is safe for concurrent access.
-func (s *server) ConnectToPeer(addr *lnwire.NetAddress, perm bool) error {
+func (s *server) ConnectToPeer(
```

Comment:
Formatting on a few functions is still a bit off, you can leave the first few params in inline then just wrap at 80.

Reply:
changed.